SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -188,6 +192,19 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         }
     }
 
+    /**
+     * This delegate method is available on iOS 11 and above. Call the completion handler once the
+     * notification payload is passed to the `TwilioVoice.handleNotification()` method.
+     */
+    func pushRegistry(_ registry: PKPushRegistry, didReceiveIncomingPushWith payload: PKPushPayload, for type: PKPushType, completion: @escaping () -> Void) {
+        NSLog("pushRegistry:didReceiveIncomingPushWithPayload:forType:completion:")
```

Comment:
General Swift style comment, should't we use `print("pushRegistry:didReceiveIncomingPushWithPayload:forType:completion:")` instead?